components/genesys-email-flow/modules/email_flow/main.tf

```diff
@@ -20,12 +20,12 @@ resource "null_resource" "deploy_files" {
 
   #This needs to be python or python3 depending on your OS installation. This was tested using 3.8 of Python
   provisioner "local-exec" {
-    command = " /usr/local/bin/python3 ${path.module}/scripts/manage_flow.py CREATE ${var.genesys_email_domain} ${var.genesys_email_domain_region}"
+    command = " /usr/local/bin/python3 ${path.module}/scripts/manage_flow.py CREATE"
```

Comment:
Hey @PrinceMerluza 

This looks really good.  What I would do here is replace the local-exec that calls the Archy flow with the new flow component.  This would get rid of the need for the Python script completely.  Remember, the Python script is nothing more than a wrapper on the Archy CLI.  Get rid of the Archy flow and we get rid of the need for the Python code.

When I originally wrote this ticket the Archy Flow component was not out of Beta yeta.

Thanks,
    John Carnell

Reply:
Hi John,

Oh right I forgot we now have flow for cx as code. I replaced the Python/Archy with CX as Code and also removed references to Archy in the blueprint doc and comments.